uber/model_checks.py

```diff
@@ -56,6 +56,18 @@ def dealer_description(group):
 
 
 @prereg_validation.Group
+def dealer_categories(group):
+    if group.tables and not group.categories:
```

Comment:
Oh, yea, huh. I sometimes forget that we have a `is_dealer` property for groups -- most of the codebase just checks if the group has tables. We should probably at some point actually change those checks.